ui/src/app/utils/index.tsx

```diff
@@ -42,16 +41,16 @@ export function isBlank(toTest: String): boolean {
   }
 }
 
+export const REGISTERED_TIER_SHORT_NAME = 'registered';
```

Comment:
Is it worth it to put this in an enum? Something that would be accessed in the code like `accessTierShortName.REGISTERED`

This might make it nicer to run functions on later on to determine what state the access tiers are in.

Reply:
good idea.  added and moved this check to a new file utils/access-tiers.tsx